### DISCUSSION TITLE
security about Allure

### DISCUSSION THREAD

**Discussion by pierrehub2b** [allure-framework/allure2#1621] [General Discussion]
Hello,

Is it safe to use Allure ? is there any relations between Yandex and Qameta companies ?
I am really affraid to use russians product or components now.

Can the community give me some advices about security when using Qameta librairies please ?

Pierre

**Comment by Avaddon-tfa** [comment]
Hey there! 

Allure 2 is absolutely safe to use. 

Though Allure was initially created in Yandex, more than 6 years ago Dmitry Baev created Allure 2, the software we are discussing. Allure 2 is completely standalone and open-sourced piece of software.

By now, Allure is maintained by Qameta Software, an Estonian startup, under [Apache-2.0 License](https://github.com/allure-framework/allure2/blob/master/LICENSE).
